hstest/django_test.py

```diff
@@ -18,11 +23,19 @@ class DjangoTest(StageTest):
     def run(self):
         if self.process is None:
             self.__find_free_port()
+            self.__prepare_database()
             self.process = subprocess.Popen([
-                self.file_to_test,
+                sys.executable, self.file_to_test,
                 'runserver', self.port, '--noreload',
             ])
 
+    def __prepare_database(self):
+        if os.path.exists(EMPTY_DATABASE):
+            shutil.copyfile(EMPTY_DATABASE, TEST_DATABASE)
+            os.environ['HYPERSKILL_TEST_DATABASE'] = TEST_DATABASE
+            migrate = subprocess.Popen([sys.executable, self.file_to_test, 'migrate'])
+            migrate.wait()
```

Comment:
Maybe check if the process has gone wrong? I use the following function to run processes. Stdout and stderr can contain useful information in case of something wrong would happen. You can define this function here, I'll move it to some utility file afterwards.
```python
def run_process(*args, need_pipes: bool = False):
    proc = subprocess.Popen(
        args,
        stdout=subprocess.PIPE if need_pipes else None,
        stderr=subprocess.PIPE if need_pipes else None
    )
    exit_code = proc.wait()
    if need_pipes:
        stdout = proc.stdout.read().decode().strip()
        stderr = proc.stderr.read().decode().strip()
        return exit_code, stdout, stderr
    else:
        return exit_code, None, None
```

Reply:
We can do it by making "send a report" situation waiting for a code and raising some exception with `stderr` output.

I add UnexpetedError exception for such cases. Is this OK?

Reply:
Yes, it's OK